nncf/sparsity/schedulers.py

```diff
@@ -236,3 +239,28 @@ def current_sparsity_level(self):
     @property
     def target_sparsity_level(self):
         return self.sparsity_levels[-1]
+
+
+@SPARSITY_SCHEDULERS.register("per_layer")
+class PerLayerSparsityScheduler(SparsityScheduler):
```

Comment:
Is there a function in the `SparsityScheduler` to set the sparsity level for the all model as for each layer?

Reply:
No, there isn`t.